source/reference/minio-mc/mc-replicate-update.rst

```diff
@@ -213,14 +213,20 @@ Parameters
    :optional:
 
    Specify the credentials, destination deployment, and bucket of the remote location.
-   Value may be either location based (IP or URL) or path based.
+   Value may be an alias/bucket, location based (IP or URL), or path based.
```

Comment:
same w.r.t. linking to alias docs

Reply:
Sidebar, we have examples for everything but path based. Should we add those and tab out the example?